setup.py

```diff
@@ -43,12 +43,10 @@
         "pyproj>=2",
         "geopandas",
         "cdsapi",
-        "pytest",
-        "black",
-        "pre-commit",
     ],
     extras_require={
-        "docs": ["numpydoc", "sphinx", "sphinx_rtd_theme", "nbsphinx", "nbsphinx-link"]
+        "docs": ["numpydoc", "sphinx", "sphinx_rtd_theme", "nbsphinx", "nbsphinx-link"],
+        "dev": ["black", "pre-commit", "pytest"],
```

Comment:
```suggestion
        "dev": ["pre-commit", "pytest"],
```

`black` should not be required as dependency (`pre-commit` takes care of black magic on its own).

Reply:
correct!